### DISCUSSION TITLE
Final Fantasy 2

### DISCUSSION THREAD

**Discussion by nl255** [jserodio/rg351p-ppsspp-settings#8] [New games]
Since there haven't been any updates on this I would like to add that neither standalone version makes it playable on the world map.  I have also tried the cores by KMFDManic from the PlayStation mini/classic and while km_xtreme_amped works the performance is still poor.  Regular km_xtreme has major graphical issues that render it unplayable.

*Note that the KMFDManic cores (found here https://github.com/KMFDManic/NESC-SNESC-Modifications/releases ) need to be decompressed before they will work on the rg351, just rename them to .so.xz and use 7zip to decompress them.

**Comment by jserodio** [comment]
Thanks for the heads up.
I'd like to throw some hope here. Not everything is lost.
There have been some conversations with Rockchip and they agreed in adding Vulcan support.
351elec has already made some changes and hopefully in 2.0 we might be able to select Vulcan instead OpenGL in the PPSSPP standalone emulator.

We're not sure yet if this will improve anything, but it's said that Vulcan performs quite better in low end devices. We'll see 😜

**Reply by nl255** [reply]
Hopefully that will help but apparently it struggles even on fairly high end computers.  It may be that FF2 is just so poorly programmed that only a rom/iso hack can fix it.  There is precedent for that as the sound restoration hack for the GBA version of FF6 also includes a framerate fix.
